pytest.ini

```diff
@@ -1,8 +1,15 @@
 [pytest]
-addopts=--doctest-modules --doctest-glob=pkg_resources/api_tests.txt -r sxX
 norecursedirs=dist build .tox .eggs
-doctest_optionflags=ELLIPSIS ALLOW_UNICODE
-filterwarnings =
+addopts=
+	--doctest-modules
+	--doctest-glob=pkg_resources/api_tests.txt
+	-r sxX
+doctest_optionflags=ALLOW_UNICODE ELLIPSIS
+# workaround for warning pytest-dev/pytest#6178
+junit_family=xunit2
```

Comment:
Do you know that xunit2 drops some of the fields available in xunit1? I've learned that using xunit1 allows representing the test results better.

Reply:
I was unaware. Thanks for the reference... and thanks for filing the issue upstream. My instinct is to rely on the degraded behavior of the newer system either until it's improved or until it's deprecated in favor of the prior implementation. Falling back to xunit1 adds technical debt that will likely require eventually switching to xunit2. I'd like to avoid that extra work across these projects.

And more importantly, Setuptools (and probably many projects) already are emitting xunit2 as part of the Pytest 6 release, so this change is a no-op compared to prior runs, so I'm going to defer this as a concern outside this PR.

Reply:
it sounds unlikely that junit2 will get that metadata because the underlying standard doesn't have this info